final/pkg/database/database.go

```diff
@@ -0,0 +1,40 @@
+package database
+
+import (
+	"github.com/NikhilSharmaWe/gomodule/final/pkg/config"
+
+	"github.com/jinzhu/gorm"
+)
+
+var db *gorm.DB
+
+type Book struct {
+	gorm.Model
+	Name        string `json:"name"`
+	Author      string `json:"author"`
+	Publication string `json:"publication"`
+}
+
+func init() {
+	config.Connect()
+	db = config.GetDB()
+	db.AutoMigrate(&Book{})
+}
+
+func (b *Book) CreateBook() *Book {
+	db.NewRecord(b)
+	db.Create(&b)
+	return b
+}
+
+func GetAllBooks() []Book {
+	var Books []Book
+	db.Find(&Books)
```

Comment:
This function might result in error sometimes, need to handle it. Same for all the DB functions.